.github/workflows/fetch.yaml

```diff
@@ -48,7 +48,11 @@ jobs:
           branch: fetch-modules
           delete-branch: true
           title: "Found new managed modules references"
-          body: "New managed modules references found. Please review."
+          body: |
+            New managed modules references found. Please review.
+            
+            - [ ] `envoyproxy/envoy` has updates, I have checked https://github.com/bufbuild/modules/issues/591
+            - [ ] `googlecloudplatform/bq-schema-api` has updates, I have checked https://github.com/bufbuild/modules/issues/592
```

Comment:
not sure if I understand this correctly, is this mean if the update for this 2 modules has a semver release, we shouldn't merge the update, instead we should move to sync with semver?

Reply:
Correct, we are waiting for _any_ semver release from `bq-schema-api`, and a _MINOR_ release from `envoy`. When that happens, we should not merge the regular update but instead switch those to SemVer.